### DISCUSSION TITLE
JetBrains plugin shows no access while the account has free access

### DISCUSSION THREAD

**Discussion by HarukaMa** [community/community#19129] [Copilot Conversations]
My account has free Copilot access (thanks btw!), however I can't use it in my IDEs:

![image](https://user-images.githubusercontent.com/861659/174949219-6c4f061e-69bd-4bea-8bfb-d0c798a39a89.png)

![image](https://user-images.githubusercontent.com/861659/174949279-23ba910b-01d9-478f-8d91-8b22b87beeb4.png)

I've suspected some state syncing in progress at first, but it's been several hours and I still can't use Copilot.

The log simply says:

```
INFO - #copilot - [INFO] [auth] [2022-06-22T05:20:14.656Z] Invalid copilot token: missing token: 403 
```

**Comment by CTTY-NUL** [comment]


**Reply by HarukaMa** [reply]
That link just redirects me to the account billing page, and I don't really have the option to cancel the "plan" there.

**Reply by HarukaMa** [reply]
nope, still the same redirection on my end.

**Comment by melvincwng** [comment]
@HarukaMa, try raising a ticket and contact [GitHub Support](https://support.github.com/) to resolve this.

Hope this helps!

**Reply by HarukaMa** [reply]
![image](https://user-images.githubusercontent.com/861659/174951227-626d648e-5687-4fbb-aa2a-93bf5f04de22.png)

Unfortunately, I'm not even sure which link is appropriate for such issue.

EDIT:

never mind, found the correct "contact us" button in one of those categories, now trying to fill in the details.

**Comment by vvsotnikov** [comment]
Same for me

**Comment by lcarcamo1526** [comment]
I got the same error.

**Comment by cncrx** [comment]
same...

**Comment by VapidLinus** [comment]
I had to go to this page and complete my settings:

https://github.com/settings/copilot

**Reply by vvsotnikov** [reply]
Didn't help in my case :(

**Reply by lcarcamo1526** [reply]
It works for me!
